torchao/prototype/float8nocompile/float8nocompile_linear_utils.py

```diff
@@ -45,10 +46,30 @@ def convert_to_float8_nocompile_training(
         config = Float8LinearConfig()
 
     from_float = lambda m: Float8LinearNoCompile.from_float(
-        m, config=config, kernel_algo=kernel_algo
+        m,
+        config=config,
+        kernel_algo=kernel_algo,
+        no_precompute_for_backward=no_precompute_for_backward,
     )
     return swap_linear_layers(
         module,
         from_float,
         module_filter_fn=module_filter_fn,
     )
+
+
+def no_precompute_for_backward_every_nth_layer(model: nn.Module, n: int):
```

Comment:
I don't think I would land this here, I'd instead just ask the user to implement this on their own.  This is more of a training loop / AC design kind of thing.

Reply:
I see, yeah I orginally had this in torchtitan training code but felt like this might be a better home for it. I can remove it from this PR.